### PR TITLE
W-10619592: implement the supportsStreaming() method for Mule Operations

### PR DESCRIPTION
**Pull request by mbuchwald**

### CODE REVIEW

**Comment by mbuchwald** [comment]
--validate

File: modules/extensions-mule-support/src/test/java/org/mule/runtime/module/extension/mule/internal/operation/MuleOperationSupportsStreamingTestCase.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.api.extension.ExtensionManager;
+import org.mule.runtime.core.api.streaming.iterator.ConsumerStreamingIterator;
+
+import org.junit.Test;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class MuleOperationSupportsStreamingTestCase extends MuleArtifactFunctionalTestCase {
```
Comment (asanguinetti) [inline modules/extensions-mule-support/src/test/java/org/mule/runtime/module/extension/mule/internal/operation/MuleOperationSupportsStreamingTestCase.java:26]: Add Allure annotations, for the test case and for the tests.

**Review by asanguinetti — CHANGES_REQUESTED** [review]
Besides the requested changes, was `modules/salida.txt` commited by mistake?

**Review by asanguinetti — APPROVED** [review]

**Review by asanguinetti — APPROVED** [review]

**Comment by mbuchwald** [comment]
--validate

**Pull request merged**